src/Resources/translations/CmfSonataAdminIntegrationBundle.de.xliff

```diff
@@ -287,8 +284,7 @@
             <!-- $Core -->
             <trans-unit id="form.tab_publish">
                 <source>form.tab_publish</source>
-                <!-- TODO: Translate -->
-                <target>Publish Settings</target>
+                <target>Einstellungen zum Veröffentlichen</target>
```

Comment:
So i did,

both words mean the same. `publizieren` as a kind of hipp english alike version of `veröffentlichen`. And even Wordpress translates it `veröffentlichen`.